bin/resty

```diff
@@ -63,6 +70,7 @@ GetOptions("c=i",             \(my $conns_num),
            "http-include=s",  \@http_includes,
            "shdict=s",        \@shdicts,
            "I=s@",            \(my $Inc),
+           "l=s",             \&require_lua_lib,
```

Comment:
Why not just use a scalar Perl variable as a flag and then test this flag using an `if` statement afterwards? Just like all the existing options' handling?

Reply:
It can be supplied several times and it depends on the order with `-e`.

So for example `-e print(1) -l foo -e print(2) -l bar` will execute: `print(1);require('foo');print(2);require('bar')`.

Because of that order I think it is easier just to add to the chunk scalar so it is executed with the rest of the chunks in correct order. 

But I don't know Perl, so maybe you are thinking something else.

Reply:
@mikz OK, you need to use `\@libs` there, just like how we implement the `--http-include` and `--shdict` options. Will you make the change to be more consistent? Thanks!

Reply:
@mikz Sorry, I mean `\@lua_libs` or `\@preloaded_lua_libs`.

Reply:
But how you ensure they are interleaved with the lua chunks? The lua libs would be in order relative to itself but not to the lua chunks by -e. It needs to perserve order from the cli.

Reply:
@mikz OK, I see what you are saying. Thanks for the explanation.